IKOR-GmbH/sip-framework
# Maven install documentation added

# Description

Documentation updated with instructions how to clone project and install local artifacts before executing archetype command.

Fixes # 26

## Type of change

Please delete options that are not relevant.

# How Has This Been Tested?

- Delete everything from \.m2\repository\de\ikor\sip directory
- Execute all commands described added with this PR
- Run archetype command with version 1.0.0
- Expected: archetype generates new adapter

# Checklist:
